resources/css/common.css

```
@@ -10,59 +9,93 @@
 --bg1:#fff;
 --bg2:#FCFCFC;
 --bg3:#f7f7f7;
---bg4:var(--point1);
---bg5:#CFE5FF;
+--bg4:var(--primary100);
+--bg4_hover:var(--primary200);
+--bg4_act:var(--primary300);
+--CoolGray:#E6F2FF;
```

💊 제안
한 레포안에서는 동일한 명명규칙을 공유해주세요~

```suggestion
--coolGray:#E6F2FF;
```